api/views/healthCheck/healthCheck.py

```diff
@@ -403,6 +404,147 @@ def health_check_yang_search_admin():
         return make_response(jsonify(error_response(service_name, err)), 200)
 
 
+@app.route('/yangcatalog', methods=['GET'])
+def health_check_yangcatalog():
+    service_name = 'yangcatalog'
+    status = 'running'
+    message = 'All URLs responded with status code 200'
+    additional_info = []
+
+    urls = ['http://yangcatalog.org',
+            'http://www.yangcatalog.org',
+            'https://yangcatalog.org',
+            'https://www.yangcatalog.org',
+            'http://yangvalidator.com',
+            'http://www.yangvalidator.com',
+            'https://yangvalidator.com',
+            'https://www.yangvalidator.com',
+            'http://18.224.127.129',
+            'https://18.224.127.129',
+            'http://[2600:1f16:ba:200:a10d:3212:e763:e720]',
+            'https://[2600:1f16:ba:200:a10d:3212:e763:e720]'
+            ]
```

Comment:
make it list of tuples or dictionary that will contain url and boolean for request verify variable

Reply:
done